tests/sles4sap/hana_install.pm

```diff
@@ -89,7 +89,13 @@ sub run {
         $hdblcm = "/sapinst/DATA_UNITS/HDB_SERVER_LINUX_PPC64/hdblcm";
         $ret    = script_run "ls $hdblcm";
     }
-    die "hdblcm is not in $hdblcm" unless (defined $ret and $ret == 0);
+    # hdblcm path can be different depending on the type of installation master in use
+    if ($ret) {
+        $hdblcm = "/sapinst/SAP_HANA_DATABASE/hdblcm";
```

Comment:
I would prefer to figure out the path where the binary is expected first, and then check once. But I won't complain, since it's outside my domain :)

Reply:
Yes, I thought about issuing a find or something more clever to find the command, but (just as an example), a find for `hdblcm` in Hana 2 SPS03's installation master returns 2 different and unrelated files. I need to be sure to be calling the `hdblcm` that installs the Hana Database Server, which for now comes in either of the paths in the code. I don't discard that there could be more paths, and that I would need to keep this in an array or something, but figured this could be enough for the time being.

Reply:
I think I will add a variable for this to clean up the code a bit.